src/Control/Email/Email.php

```diff
@@ -584,13 +591,16 @@ public function setPriority($priority)
     }
 
     /**
+     * @deprecated 4.12.0 Will be replaced with Symfony\Component\Mime\Email::attachFromPath()
```

Comment:
```suggestion
     * @deprecated 4.12.0 Use attachFromPath() instead
```
Putting `Symfony\Component\Mime\Email` there makes it look like they have to use an entirely different class!
Also the "use [other method] instead" syntax has been used in the past, so we should continue with that syntax.

Reply:
> Also the "use [other method] instead" syntax has been used in the past

The problem here is that attachFromPath() doesn't exist in `4`.  If we say "Use attachFromPath() instead" then providing instructions to replace calls to this method with a non-existent method.

So we're forced to use "Will be replaced with" syntax here, which isn't great.  We could say something else like "In future versions use attachFromPath() which is currently unimplemented"? But that sounds worse.

I think the way I already have this is probably the least worst

Reply:
That's a good point - In that case just removing the FQCN will be okay here.